internal/controller/install/armadaserver_controller.go

```diff
@@ -678,3 +687,21 @@ func createServerPrometheusRule(server *installv1alpha1.ArmadaServer) *monitorin
 		},
 	}
 }
+
+type ArmadaServerConfig struct {
```

Comment:
Can you reuse the `CommonApplicationConfig` as it already contains the `grpc` configuration block?
As currently this defines component-specific config structs which have a common field.

Here is the struct - https://github.com/armadaproject/armada-operator/blob/9e41715dd4bbae49d68d2c49a7961a36d32d2561/internal/controller/builders/config.go#L16

Reply:
Definitely. I did not see that